### DISCUSSION TITLE
How to nested ListBox/GridList components.

### DISCUSSION THREAD

**Discussion by brenotx** [adobe/react-spectrum#6016] [Q&A] [answered]
Hey there! I'm curious about which component from `react-aria` would be best for creating nested expanded lists. I have tried to nest ListBox but it doesn't seem to work. Eg.:

```jsx
<ListBox
  aria-label="Animals"
  items={options}
  selectionMode="multiple"
>
  {item => (
    <ListBox
      aria-label="Animals"
      items={item.subItem}
      selectionMode="multiple"
    >
      {subItem => <ListBoxItem>{subItem.name}</ListBoxItem>}
    </ListBox>
  )}
</ListBox>
```
Here is an illustration of what I'm trying to accomplish:

https://github.com/adobe/react-spectrum/assets/2152969/849079bd-afa2-4d6e-8cff-a96c2dce3428

**Comment by snowystinger** [comment, marked as answer]
That looks more like a Tree, which we are currently working on. You could checkout the PR for it https://github.com/adobe/react-spectrum/pull/5756 and you can try it out in nightlies
